cli/kubectl-hyperkube

```diff
@@ -3,13 +3,17 @@
 import click
 import json
 from pathlib import Path
-from pprint import pprint
+
 import requests
 import sys
 import yaml
 import os
 
 
+def hkc_print(message, *args, **kwargs):
```

Comment:
Shouldn't it be declared in util.py ?

Reply:
That would be nice but the cli tool is currently stand alone and util.py is not packaged with it. I think that would be another pull request.